### PR TITLE
Update feature template

### PR DESCRIPTION
**Pull request by mbroz2**

### CODE REVIEW

**Comment by mbroz2** [comment]
fyi @tbitonti

**Comment by mbroz2** [comment]
@cbridgha @richardherget fyi

**Review by samwatibm — APPROVED** [review]
Looks good!

File: .github/ISSUE_TEMPLATE/feature.md [hunk 1]
```diff
@@ -1,104 +1,202 @@
...
+
+- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+## **Design** (Complete Before Development Starts)
+
+Design preliminaries determine whether a formal design, which will be provided by an Upcoming Feature Overview (UFO) document, must be created and reviewed.  A formal design is required if the feature requires any of the following: UI, Serviceability, SVT, Performance testing, or non-trival documentation/ID.
+
+### **Design Preliminaries**
+- [ ] UI requirements identified. (Owner and UI focal point)
+- [ ] ID requirements identified. (Owner and ID focal point) ([Karen Deen](https://github.com/chirp1))
+   - Rerfer to [Documenting Open Liberty](https://github.com/OpenLiberty/open-liberty/wiki/Documenting-Open-Liberty).
```
Comment (tevans78) [inline .github/ISSUE_TEMPLATE/feature.md]:
```suggestion
   - Refer to [Documenting Open Liberty](https://github.com/OpenLiberty/open-liberty/wiki/Documenting-Open-Liberty).
```

**Pull request merged**